src/zip.ts

```diff
@@ -89,6 +96,8 @@ export const zipFunctions = async function (
         stat: func.stat,
       })
 
+      zippedFunctionOutput.push({ name: func.name, config: func.config, featureFlags: functionFlags })
```

Comment:
Is there a reason to buffer the logs as opposed to emitting one per function?

Reply:
I think emitting separate logs would make the ZISI code a bit simpler, as you don't need to buffer the messages, and would protect us a bit from super-long log lines. It's true that these can get intertwined with other logs, but I personally don't think that's a huge deal. You can always adjust your Humio query to filter them out.

Reply:
Done. This is what an individual log looks like now:

```
Function details: {
  "name": "hello-world",
  "config": {
    "includedFilesBasePath": "/Users/jackie/src/playground/edge-functions-examples",
    "zipGo": true
  },
  "featureFlags": {
    "buildRustSource": false,
    "parseWithEsbuild": false,
    "traceWithNft": false,
    "zisi_pure_esm": false,
    "zisi_pure_esm_mjs": false,
    "project_deploy_configuration_api_use_per_function_configuration_files": true
  }
}
```

Reply:
Is there anything more we can do with ZISI logging that could help with how long bundling takes?